c10/test/util/string_view_test.cpp

```diff
@@ -366,15 +330,6 @@ static_assert(string_view("hello").substr(0, 100) == string_view("hello"), "");
 static_assert(string_view("hello").substr(1, 100) == string_view("ello"), "");
 static_assert(string_view("hello").substr(5, 100) == string_view(""), "");
 
-TEST(StringViewTest, whenCallingSubstrWithPosOutOfRange_thenThrows) {
-  expectThrows<std::out_of_range>(
-      [] { string_view("hello").substr(6); },
-      "basic_string_view::substr parameter out of bounds. Index: 6, size: 5");
-
-  expectThrows<std::out_of_range>(
-      [] { string_view("hello").substr(6, 0); },
-      "basic_string_view::substr parameter out of bounds. Index: 6, size: 5");
-}
```

Comment:
So this is what, standard string view has less error checking for out of bounds stuff?

Reply:
Different compilers throw different exception messages for std::out_of_range although the standard requires std::out_of_range.

Reply:
ok. I guess it's not a big deal to stop testing the std class anyway